Tools/find-includes-cycles.py

```diff
@@ -4,23 +4,17 @@
 Run this script from Source/Core/ to find all the #include cycles.
 '''
 
-import subprocess
+from typing import Iterator
+from pathlib import Path
 
-def get_local_includes_for(path):
-    lines = open(path).read().split('\n')
-    includes = [l.strip() for l in lines if l.strip().startswith('#include')]
-    return [i.split()[1][1:-1] for i in includes if '"' in i.split()[1]]
+def get_local_includes_for(p: Path) -> Iterator[str]:
+    with p.open() as f:
+        for l in f:
+            l = l.strip()
+            if l.startswith("#include") and '"' in l:
+                yield l.split()[1].strip(' "')
```

Comment:
Let's not make it harder to understand (especially for people unfamiliar with python) and not shorten variable names.

```suggestion
        for line in f:
            line = line.strip()
            if line.startswith("#include") and '"' in line:
                yield line.split()[1].strip(' "')
```